src/props.c

```diff
@@ -361,6 +361,9 @@ int simple_propfind(const char *path, size_t depth, time_t last_updated, props_r
         XML_SetCharacterDataHandler(parser, characterDataHandler);
         curl_easy_setopt(session, CURLOPT_WRITEDATA, (void *) parser);
         curl_easy_setopt(session, CURLOPT_WRITEFUNCTION, write_parsing_callback);
+        curl_easy_setopt(session, CURLOPT_TIMEOUT, 0);
+        curl_easy_setopt(session, CURLOPT_LOW_SPEED_LIMIT, 16384);
```

Comment:
I would drop this to a lower number. This currently sets the rate around a megabyte per minute, but that would be hard to hit if there's an initial lead time of 60-90 seconds to send the first byte. I would consider setting it extremely low; it's not like we're going to intentionally drip data to clients, and malfunctions generally cause the server to close the connection or send nothing at all.